jmix-flowui/flowui/src/test/groovy/data_binding_html_container/HtmlContainerReadonlyDataBindingTest.groovy

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package data_binding_html_container
+
+import com.vaadin.flow.component.html.Div
+import data_binding_html_container.view.DataBindingHtmlContainerView
+import io.jmix.core.DataManager
+import io.jmix.core.MetadataTools
+import io.jmix.core.SaveContext
+import io.jmix.flowui.data.binding.HtmlContainerReadonlyDataBinding
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.jdbc.core.JdbcTemplate
+import test_support.entity.Animal
+import test_support.entity.Zoo
+import test_support.spec.FlowuiTestSpecification
+
+import java.util.stream.Collectors
+
+@SpringBootTest
+class HtmlContainerReadonlyDataBindingTest extends FlowuiTestSpecification {
```

Comment:
I think we should write one more test for a HTML component that only has a `property` attribute, and this HTML component should be located in a `formLayout` in which the `dataContainer` is specified